modules/terraform/validate_struct.go

```diff
@@ -0,0 +1,103 @@
+package terraform
+
+import (
+	"fmt"
+	"path"
+	"path/filepath"
+
+	"github.com/gruntwork-io/terratest/modules/collections"
+	"github.com/mattn/go-zglob"
+)
+
+type ValidationOptions struct {
+	// The target directory to recursively search for all Terraform directories (those that contain .tf files)
+	// If you provide RootDir and do not pass entries in either IncludeDirs or ExcludeDirs, then all Terraform directories
+	// From the RootDir, recursively, will be validated
+	RootDir string
+	// If you only want to include certain sub directories, add them as paths relative to RootDir. For example, if the
+	// RootDir is /home/project and you want to only include /home/project/examples, add /home/project/examples here
+	// Note that while the struct requires full paths, you can pass relative paths to the NewValidationOptions function
+	// which will build the full paths based on the supplied RootDir
+	IncludeDirs []string
+	// If you want to explicitly exclude certain sub directories, add them as paths relative to RootDir. For example, if the
+	// RootDir is /home/project and you want to include everything EXCEPT /home/project/modules, add
+	// /home/project/modules to this slice. Note that ExcludeDirs is only considered when IncludeDirs is not passed
+	// Note that while the struct requires full paths, you can pass relative paths to the NewValidationOptions function
+	// which will build the full paths based on the supplied RootDir
+	ExcludeDirs []string
+}
+
+// NewValidationOptions returns a ValidationOptions struct, with override-able sane defaults
+func NewValidationOptions(rootDir string, includeDirs, excludeDirs []string) (ValidationOptions, error) {
```

Comment:
NIT: Add comment indicating the `includeDirs` and `excludeDirs` are relative to the `rootDir`. I know you already mention this in the struct docs, but it is useful to mention it here was well, since some users might only look at the godocs for this function.

Reply:
Makes sense - done in 95eb0a9